### DISCUSSION TITLE
How to set the default grub menu timeout?

### DISCUSSION THREAD

**Discussion by easpeagle** [cobbler/cobbler#3438] [Q&A]
Hey Folks,

I'm trying and failing to find out to override the default grub menu timeout. It gets set to 1 second... and I'd like to make that a little longer. Where can this be done on a per distro/profile/system basis?

Thanks!

**Comment by SchoolGuy** [comment]
There is a template in `/etc/cobbler/boot_loader_conf/grub.template`. In there you should be able to set the timeout for GRUB. In case that is not enough you also have in the same folder a file called `grub_submenu.template` that you can adjust. Play a little with the templates they are there to be adjusted to your needs.

With 3.4.0 I will overhaul the template loading and as such expect changes in that area.

Edit: Don't forget to ask follow-up questions if there are some and to mark the appropriate answer as the correct one.

**Reply by wangxu186189** [reply]
For cobbler v3.2.2,  u can adjust the timeout time by modifying this configuration file : /var/lib/cobbler/grub_config/grub/grub.cfg   (line 15 and line 16)   I hope I can help you
